cdq-analysis/main.go

```diff
@@ -80,7 +80,13 @@ func main() {
 		Log:          log,
 		CreateK8sJob: createK8sJob,
 	}
-	pkg.CloneAndAnalyze(k8sInfoClient, gitToken, namespace, name, contextPath, devfilePath, dockerfilePath, URL, Revision, DevfileRegistryURL, isDevfilePresent, isDockerfilePresent)
+
+	cdqInfo := &pkg.CDQInfoClient{
+		DevfileRegistryURL: DevfileRegistryURL,
```

Comment:
I will remove the dockerfile and devfilepath  since these are now computed in the FindValidDevfile/Dockerfile functions.  Since we don't want external values to be passed in, I'll also update the fields in the CDQClientInfo struct to private scope and introduce a new getter to retrieve the values